CMakeLists.txt

```diff
@@ -26,9 +26,11 @@ Various components may fail to compile with default configuration options, and s
 include(cmake/versions.cmake) # Required for OMR_VERSION 
 
 project(omr VERSION ${OMR_VERSION} LANGUAGES CXX C)
+set(OMR_ROOT ${CMAKE_CURRENT_LIST_DIR} CACHE INTERNAL "Root of OMR")
```

Comment:
Not needed. Can just use `OMR_SOURCE_DIR` 
see https://cmake.org/cmake/help/v3.9/variable/PROJECT-NAME_SOURCE_DIR.html